README.md

```diff
@@ -26,12 +31,14 @@ let package = Package(
     ],
     dependencies: [
         ...
+        // Danger
+        .package(name: "danger-swift", url: "https://github.com/danger/swift.git", from: "3.0.0"), // dev
         // Danger Plugins
-        .package(url: "https://github.com/f-meloni/danger-swift-xcodesummary", from: "0.1.0") // dev
+        .package(name: "DangerXCodeSummary", url: "https://github.com/f-meloni/danger-swift-xcodesummary", from: "1.2.1"), // dev
         ...
     ],
     targets: [
-        .target(name: "DangerDependencies", dependencies: ["Danger", "DangerXCodeSummary"]), // dev
+        .target(name: "DangerDependencies", dependencies: ["danger-swift", "DangerXCodeSummary"]) // dev
```

Comment:
I think that could potentially remove some optimisations SPM would do here, given you only need the `Danger` library as dependency not the whole danger-swift, you don't need to build it all, then I would probably remove all the `name: "` change, at least for the `danger-swift` part.